phac-nml/irida-plugin-example
# Metadata API update

This PR implements the new API used for updating metadata from IRIDA (https://github.com/phac-nml/irida/pull/576) and the updates to the IRIDA pipeline plugin system (https://github.com/tom114/irida/pull/38).

Due to a change to the Secure Hash tool in Galaxy, it also updates the example plugin to the new version of this tool.

To test,
1. Install the IRIDA branch from the Plugin 1.1 PR https://github.com/tom114/irida/pull/38/ and install with `mvn clean install -DskipTests` on the IRIDA repo.
2. Build this plugin with `mvn clean package`.
3. Install this plugin in the `/etc/irida/plugins` directory.
4. Run Galaxy and the IRDIA you checked out above (tested on a fresh install of Galaxy 20.05).
5. Run the pipeline plugin and verify it both runs and writes back the info to the metadata tables.
